badge_server/main.py

```diff
@@ -76,6 +76,21 @@ def _get_self_compatibility_dict(package_name: str) -> dict:
     return result_dict
 
 
+def _get_other_package_from_set(name: str,
+                                package_set: frozenset) -> package.Package:
+    """Returns the other package.
+
+    Args:
+        name: a string, the given package name
```

Comment:
You don't have to duplicate the type annotation.
```suggestion
        name: The name of the package not to return.
```

Reply:
done